source/chapters/setup.rst

```diff
@@ -85,86 +137,111 @@ Laptop and an External USB Soundcard
 
 The most common setup for DJing is a laptop and a soundcard with 2 stereo
 outputs (4 channels). The first stereo output (channels 1-2) is used as the
-**Master out** and supplies the main mix to the room. With the 2nd stereo output
-(channels 3-4) you can :term:`cue <cueing>` and
-:ref:`preview <djing-previewing-tracks>` the next track in your headphones. If
-your soundcard does not have 2 stereo outputs, you can use multiple soundcards
-to achieve the same setup.
-
-The depicted configuration above uses two soundcards. This setup is the bare
+master output and supplies the main mix to the audience. With the second stereo
+output (channels 3-4) you can :term:`cue <cueing>` and :ref:`preview 
+<djing-previewing-tracks>` the next track in your headphones. If
+your soundcard only has one stereo output, you can use multiple soundcards
+to achieve a similar setup. Most DJ :term:`MIDI controllers <MIDI controller>`
+above the most basic models include a soundcard with 2 stereo outputs
+(see :ref:`setup-controller-and-external-card` for details).
+
+The configuration depicted above uses two soundcards. This setup is the bare
 minimum for serious DJing. Without any extra gear, the DJ can control Mixxx via
 mouse and keyboard. For more information see :ref:`controlling mixxx`.
 
-**Using an external multi-channel soundcard**
+**Using two soundcards (built-in + external)**
 
 #. Open :menuselection:`Preferences --> Sound Hardware`
 #. Select the :guilabel:`Output` tab
-#. From the :guilabel:`Master` drop-down menus, select the external soundcard,
+#. From the :guilabel:`Master` drop-down menu, select the external soundcard,
    then :guilabel:`Channels 1-2`
-#. From the :guilabel:`Headphones` drop-down menus, select the external
-   soundcard, then :guilabel:`Channels 3-4`
+#. From the :guilabel:`Headphones` drop-down menu, select the built-in
+   soundcard, then :guilabel:`Channels 1-2`
 #. Click :guilabel:`Apply` to save the changes.
 
-**Using two soundcards (built-in + external)**
+**Using an external multi-channel soundcard**
```

Comment:
add a few sentences about the multi-card sync options we have.  "no delay" is pretty good these days, despite the "experimental" tag in the UI